.changeset/clean-flowers-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': minor
+---
+
+`Portal`'s `portal.to` now also accepts `null | undefined` which acts identically to the default prop behavior (i.e. as if `portal` was not passed). Consequently, `Dialog` & `Modal`'s `portal.to` also reflects this change.
```

Comment:
`Portal` is not public facing, so this should be rewritten to only mention public compoenents.

There are quite a few (e.g. `Popover`/`Tooltip`/`SplitButton`)

Reply:
Oops, fixed (https://github.com/iTwin/iTwinUI/pull/1786/files/c307587395bc8c3483fee14becce85822879878e..77b86d599d1c9bc923c991a5e56b4c1af28c1df2)

Reply:
Readability:

```suggestion
The `portal.to` prop in floating elements now also accepts `null | undefined` which acts identically to the default prop behavior (i.e. as if `portal` was not passed). Components affected: `Dialog`, `DropdownMenu`, `Modal`, `Popover`, `SplitButton`, and `Tooltip`.
```

Reply:
Done ([`9a7f68d` (#1786)](https://github.com/iTwin/iTwinUI/pull/1786/commits/9a7f68d7224142e9d56664ba9285af96bac70da7))